### PR TITLE
Global import ordering

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: code/src/java/gmgen/plugin/PcgSystemInitiative.java [hunk 1]
```diff
@@ -20,14 +20,15 @@
...
 
-import gmgen.plugin.dice.Dice;
 import pcgen.cdom.util.CControl;
 import pcgen.cdom.util.ControlUtilities;
 import pcgen.core.Globals;
 import pcgen.core.PCStat;
 import pcgen.core.PlayerCharacter;
 import pcgen.core.display.CharacterDisplay;
 
+import gmgen.plugin.dice.Dice;
```
Comment (thpr) [inline code/src/java/gmgen/plugin/PcgSystemInitiative.java]: Your config file currently reads gmgem not gmgen

**Comment by karianna** [comment]
@grimreaper Can you take a look at @thpr feedback?

File: code/src/java/gmgen/plugin/PcgSystemInitiative.java (hunk 1 above)
Comment (LegacyKing) [reply code/src/java/gmgen/plugin/PcgSystemInitiative.java]: I don't see this GMGem reference here... please verify?

**Comment by LegacyKing** [comment]
This looks good to me. I see no misspelled GMGen entries.

**Pull request merged**